integration/tests/verification/suite.go

```diff
@@ -25,8 +25,9 @@ type Suite struct {
 	nodeConfigs                []testnet.NodeConfig // used to keep configuration of nodes in testnet
 	nodeIDs                    []flow.Identifier    // used to keep identifier of nodes in testnet
 	ghostID                    flow.Identifier      // represents id of ghost node
-	exeID                      flow.Identifier      // represents id of execution node
-	verID                      flow.Identifier      // represents id of verification node
+	exe1ID                     flow.Identifier
+	exe2ID                     flow.Identifier
```

Comment:
Even on sealing based on result approvals, we need more than one agreeing-on execution result for a block to be sealed. Hence, we need more than one execution node.